projects/storefrontapp-e2e-cypress/cypress/e2e/vendor/asm/b2b/customer-list.e2e.cy.ts

```diff
@@ -20,6 +20,20 @@ context('Assisted Service Module', () => {
     Cypress.env('BASE_SITE', ELECTRONICS_BASESITE);
   });
   describe('ASM Customer list', () => {
+    it('Should be able to create a b2b cart via API (CXSPA-1595)', function () {
```

Comment:
Why is this case belonging to ASM,  and did not see the agent login

Reply:
Add product to the cart, then login to the agent account and check if the shopping cart icon of the user on the customer list page is clickable.

Reply:
I see, but I don't see any code relevant with agent login

Reply:
In this test, "checking custom list features", the customer list will be inspected and the following code will check if the cart is clickable. https://github.com/SAP/spartacus/blob/1732a63a1be890136b10dc00aa94b737147b0839/projects/storefrontapp-e2e-cypress/cypress/helpers/asm.ts
```
cy.log('--> start emulation by click cart');
  asm.asmOpenCustomerList();
  cy.get('cx-customer-list')
    .find('.cx-btn-cell')
    .filter('.cx-cart')
    .then(($rows) => {
      cy.wrap($rows[0]).click();
      cy.get('cx-customer-list').should('not.exist');
      cy.get('cx-add-to-saved-cart').should('exist');
    });
```

Reply:
I see, but I didn't see any code relevant with agent login, could you please tell me where did the agent login?  you mean that in another case('checking custom list features')

Reply:
Okay, I've merged it into one test